lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -63,6 +69,11 @@ sub expand_item_in_system_view_table {
     send_key('right');
 }
 
+sub go_top_in_system_view_table {
```

Comment:
table or section? it is a tree view.

Reply:
Yes, I though of naming it like something_leftpane_tree but then it would be incoherent with the naming of other functions like select_item_in_system_view_table and expand_item_in_system_view_table